deploy/charts/harvester/templates/longhorn-device-cleaner.yaml

```diff
@@ -0,0 +1,57 @@
+# This configmap is a hook to add the migratable param to the longhorn storage class
+{{ if .Values.longhorn.enabled -}}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: longhorn-loop-device-cleaner
+  namespace: {{ .Values.longhorn.namespaceOverride }}
+  labels:
+    app: longhorn-loop-device-cleaner
+spec:
+  selector:
+    matchLabels:
+      app: longhorn-loop-device-cleaner
+  template:
+    metadata:
+      labels:
+        app: longhorn-loop-device-cleaner
+    spec:
+      hostNetwork: true
+      hostPID: true
+      containers:
+      - name: longhorn-loop-device-cleaner
+        command:
+          - nsenter
+          - --mount=/proc/1/ns/mnt
+          - --
+          - bash
+          - -c
+          - |
+            while true ;
+            do
+                for device in /sys/block/loop* ; do
+                        echo "INFO: checking loop device $device"
+                        if test -f "$device/loop/backing_file"; then
+                                backingFile=`cat "$device/loop/backing_file"`
+                                # removing trailling spaces
+                                backingFile=`echo $backingFile | sed 's/ *$//g'`
+                                if [[ $backingFile == *"(deleted)" ]] ; then
+                                        loopDevice="/dev/$(echo $device | cut -d"/" -f4)"
+                                        echo "WARN: detected removed backing file $backingFile"
+                                        echo "WARN: detaching the loop device $loopDevice"
+                                        losetup -d $loopDevice
+                                fi
+                        else
+                                echo "INFO: skipping loop device $device: backing file $device/loop/backing_file doesn't exist"
+                        fi
+                done
+                sleepingTime=60
+                echo "INFO: sleeping for ${sleepingTime}s"
+                sleep $sleepingTime
+            done
+        image: alpine:3.12
```

Comment:
We could template the image name here with a values file variable. (e.g., https://github.com/harvester/harvester/blob/b20a3b47fb06b649ea5762be021372a8a728411e/deploy/charts/harvester/values.yaml#L442-L446)
The benefit is a harvester-installer build can automatically [pull needed images for air-gapped environments.](https://github.com/harvester/harvester-installer/blob/9cae7726a46e19e66f69ccb61c55cd1c56ead97a/scripts/build-bundle#L96-L109)